bin/bin/dep.ml

```diff
@@ -146,8 +146,8 @@ let process_md ~toc book_dir =
         let html = chapter ^ ".html" in
         sprintf {|(rule
   (targets %s)
-  (deps    %s)
-  (action  (run mdx output %%{deps} -o %%{targets})))|}
+  (deps    (:x %s) %%{bin:rwo-output})
```

Comment:
I think a more explicit name than `x` would probably be better!

Also, with dune, you don't need to explicit declare the dependency on `rwo-output` since it knows how to build the first argument to `run`, it understands this dependency on its own.

Reply:
Okay, thank you for this information, I'll  change that